src/loginServices.js

```diff
@@ -182,8 +182,14 @@ function dispatchLocale(url, store, tenant) {
  * @returns {Promise}
  */
 export function getLocale(okapiUrl, store, tenant) {
+  const query = [
+    'module==ORG',
+    'configName == localeSettings',
+    '(cql.allRecords=1 NOT userId="" NOT code="")'
```

Comment:
Also, why does `code` come into this?

Reply:
A config entry has up to four qualifiers (`module`, `configName`, `code`, `userId`) and a `value`. ui-tenant-settings stores its value by specifying only the `module` and `configName` qualifiers. Thus, this query seeks to retrieve the record where those qualifiers are present and the others are absent. If we don't ensure that `code` is absent, the query is open to pollution from records like 
```
{
  module: "ORG",
  configName: "localSettings",
  code: "hahahah yer in trouble now",
  value: "{locale: \"Klingon\"}"
}
```

Reply:
You are right about code-pollution, of course. I feel like an idiot for not having seen that immediately.

Reply:
Pffffft. If you're an idiot than we all are; this insufficiently-strict-query bug has been lurking since [the initial implementation in 2017](https://github.com/folio-org/stripes-core/pull/17).